ccd-definition/CaseEvent/CareSupervision/Gatekeeping-prod.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
how does this toggle work - so, existing event is available in gatekeeping, but with the additional urgent order - then after toggle, event is available in case management too?

Reply:
So there are 2 toggles for this story, the first is this one which controls the state and the CYA changes, the second is with the fixed list, the new option in the fixed list is toggled so that it won't be selectable by the user. This then allows me to not need to toggle all the other code as the sdoRouter will never have the urgent value selected until the toggled fixed lists are merged

Reply:
OK, I trust you to do the toggles properly!